spec/console_app_spec.rb

```diff
@@ -13,37 +13,39 @@
   it "initialize a 3x3, HVH game" do
     allow(game_factory_spy).to receive(:create_game).and_return(game_spy)
     expect(game_factory_spy).to receive(:create_game).and_return(game_spy)
-    expect(game_spy).to receive(:game_type).and_return(GameType::HVH)
-    game = console.create_new_game_from_options(BoardOptions::THREE_BY_THREE, GameType::HVH)
-    expect(console.game_type).to eq(GameType::HVH)
+    expect(game_spy).to receive(:game_type).and_return(GameTypeOptions::HVH)
+    game = console.create_new_game_from_options(BoardOptions::THREE_BY_THREE, GameTypeOptions::HVH)
+    expect(console.game_type).to eq(GameTypeOptions::HVH)
   end
 
   it "sets up the game and plays" do
     expect(game_factory_spy).to receive(:create_game).and_return(game_spy)
     expect(game_spy).to receive(:play_turns).and_return(board_spy)
-    console.create_new_game_from_options(BoardOptions::THREE_BY_THREE, GameType::HVH)
+    console.create_new_game_from_options(BoardOptions::THREE_BY_THREE, GameTypeOptions::HVH)
     console.play
   end
 
   it "user provides invalid game type" do
-    expect(display_spy).to receive(:ask_player_for_game_type).and_return("9", GameType::HVH)
+    expect(display_spy).to receive(:ask_player_for_game_type).and_return("9", GameTypeOptions::HVH)
     expect(display_spy).to receive(:ask_player_for_board_dimension).and_return(BoardOptions::THREE_BY_THREE)
-    expect(game_factory_spy).to receive(:create_game).with(BoardOptions::THREE_BY_THREE, GameType::HVH).and_return(game_spy)
+    expect(game_factory_spy).to receive(:create_game).with(BoardOptions::THREE_BY_THREE, GameTypeOptions::HVH).and_return(game_spy)
     console.initialize_game
   end
 
   it "user provides invalid board dimension" do
-    expect(display_spy).to receive(:ask_player_for_game_type).and_return(GameType::HVH)
+    expect(display_spy).to receive(:ask_player_for_game_type).and_return(GameTypeOptions::HVH)
     expect(display_spy).to receive(:ask_player_for_board_dimension).and_return("5", BoardOptions::THREE_BY_THREE)
-    expect(game_factory_spy).to receive(:create_game).with(BoardOptions::THREE_BY_THREE, GameType::HVH).and_return(game_spy)
+    expect(game_factory_spy).to receive(:create_game).with(BoardOptions::THREE_BY_THREE, GameTypeOptions::HVH).and_return(game_spy)
     console.initialize_game
   end
 
-  it "user provides valid game type and board size" do
-    expect(display_spy).to receive(:ask_player_for_game_type).and_return(GameType::HVH)
-    expect(display_spy).to receive(:ask_player_for_board_dimension).and_return(BoardOptions::THREE_BY_THREE)
-    expect(game_factory_spy).to receive(:create_game).with(BoardOptions::THREE_BY_THREE, GameType::HVH).and_return(game_spy)
-    console.initialize_game
+  [GameTypeOptions::HVH, GameTypeOptions::BVH, GameTypeOptions::HVB].map do |game_type|
```

Comment:
seems wasteful to do this for all game types, based on the expectations of the test. This feels like you're trying to exhaustively test for correctness, but if a new game option is added, this won't check that new type. Move complex text (with the map over a list) without any added value.